src/creature.cpp

```diff
@@ -652,11 +652,11 @@ void Creature::onDeath()
 {
 	bool lastHitUnjustified = false;
 	bool mostDamageUnjustified = false;
-	Creature* _lastHitCreature = g_game.getCreatureByID(lastHitCreature);
+	Creature* lastHitCreature = g_game.getCreatureByID(this->lastHitCreature);
```

Comment:
Done. Also renamed a few occurrences of `houseid` and `townid` to `houseId` and `townId` for consistency.